.vuepress/config.js

```diff
@@ -72,6 +72,7 @@ module.exports = {
           { text: 'GitHub', link: 'https://github.com/meilisearch/meilisearch' },
           { text: 'Slack', link: 'https://slack.meilisearch.com' },
           { text: 'Blog', link: 'https://blog.meilisearch.com/' },
+          { text: 'Datasets', link: 'https://github.com/meilisearch/datasets/' },
```

Comment:
We don't have to add this here. We link the datasets in the Quick start multiple times

Reply:
It does look a bit out of place there. I'm not a fan of providing datasets because I don't want to keep them up-to-date, but don't feel super strongly about it.

I would vote for removing it, but if you think it's helpful for users I trust your judgement 👍

Reply:
Na, I don't like it either. Just thought if we _really_ wanted to have a link, we could have it here.